config/routes.yaml

```diff
@@ -13,4 +9,4 @@ nelmio_security_report:
 
 logout:
     path: /logout
-    methods: [POST]
+    methods: [GET, POST]
```

Comment:
Is GET still required? As I know it, we send a POST request to the /logout route (by clicking the logout link). There the session is cleared. And we are redirected to the configured logout readme page.

Reply:
If it wasn't here. I don;t think so. The reason was to test the logout in de web profiler (which only uses a get)

I remove that